### DISCUSSION TITLE
pack() not found

### DISCUSSION THREAD

**Discussion by yanxke** [jfjlaros/simpleRPC#35] [Q&A]
https://simplerpc.readthedocs.io/en/latest/usage.html#exporting-class-methods

Where is pack() defined or how do I implement it?  The examples don't compile.

**Comment by jfjlaros** [comment]
This function has been renamed to `makeTuple` in 4710992a.

The documentation is in sync with the latest release, hence the confusion. The development version has moved on quite a bit since then.
